foreman_remote_execution.gemspec

```diff
@@ -14,7 +14,7 @@ Gem::Specification.new do |s|
   s.description = 'A plugin bringing remote execution to the Foreman, completing the config ' +
                   'management functionality with remote management functionality.'
 
-  s.files =            Dir['{app,config,doc,db,lib,locale}/**/*', 'LICENSE', 'Rakefile', 'README.md']
+  s.files =            `git ls-files`.split("\n")
   s.test_files =       Dir['test/**/*']
```

Comment:
You need to do the same for `test_files` and `extra_rdoc_files`, I've tested the code and it still doesn't fix the issue with extra files in `doc`